client/cli/src/commands/mod.rs

```diff
@@ -53,14 +82,41 @@ pub enum Subcommand {
 	/// Validate a single block.
 	CheckBlock(CheckBlockCmd),
 
+	/// Export state as raw chain spec.
+	ExportState(ExportStateCmd),
+
 	/// Revert chain to the previous state.
 	Revert(RevertCmd),
 
 	/// Remove the whole chain data.
 	PurgeChain(PurgeChainCmd),
 
-	/// Export state as raw chain spec.
-	ExportState(ExportStateCmd),
+	/// Generate a random node libp2p key, save it to file and print its peer ID
+	GenerateNodeKey(GenerateNodeKeyCmd),
+
+	/// Generate a random account
+	Generate(GenerateCmd),
+
+	/// Gets a public key and a SS58 address from the provided Secret URI
+	InspectKey(InspectCmd),
+
+	/// Insert a key to the keystore of a node.
+	Insert(InsertCmd),
+
+	/// Sign a message, with a given (secret) key
+	Sign(SignCmd),
+
+	/// Sign transaction from encoded Call. Returns a signed and encoded UncheckedMortalCompactExtrinsic as hex.
+	SignTransaction(SignTransactionCmd),
+
+	/// Author and sign a Node pallet_balances::Transfer transaction with a given (secret) key.
+	Transfer(TransferCmd),
```

Comment:
How does this work for chains who don't use the `pallet-balances`?

Reply:
well the requirement for using the CLI is your runtime must implement pallet-balances, although I think the logical next step is put stuff like this behind a feature flag

Reply:
If we really need this (not sure), it should be made optional in its own crate. Like the `benchmark` sub command for example.

Reply:
Does it work now? Last I knew, `subkey transfer` was really out of date.

Reply:
I would expect that it works after this pr.